packages/app-store/stripepayment/lib/PaymentService.ts

```diff
@@ -57,11 +68,67 @@ export class PaymentService implements IAbstractPaymentService {
       );
       const paymentFee = Math.round(payment.amount * payment_fee_percentage + payment_fee_fixed);
 
+      const customer = await retrieveOrCreateStripeCustomerByEmail(
+        bookerEmail,
+        this.credentials.stripe_user_id
+      );
+
```

Comment:
Setup intents must be attached to a customer. Right now we do not attach any payments to a Stripe customer